### DISCUSSION TITLE
Error after installing Apache Airflow on Linux Red Hat Enterprise Development

### DISCUSSION THREAD

**Discussion by bneal1987** [apache/airflow#21198] [Q&A]
Recently installed Apache Airflow on Linux Red Hat Enterprise 8, development version. Able to install successfully via command. 

Accessed the URL at http://localhost:8080

Went to create a new user to log in, which was successful based on the command.

When closing Firefox and re-open, we received the error attached. 

Restarted the server AFTER installing airflow and BEFORE creating the user.

Any assistance or guidance with using Red Hat / Apache Airflow and troubleshooting our issue would be greatly appreciated!

![Screenshot 2022-01-28 at 17-21-48 Screenshot](https://user-images.githubusercontent.com/82835140/151629949-21f734e2-c9cd-4399-9ade-fd3adfd0687f.png)

**Comment by bneal1987** [comment]
Looking for any thoughts, suggestions, comments, discussions, etc. on this thread. 

Thank you,

**Reply by potiuk** [reply]
You seem to use different DB in your webserver than the one that you used when you created users and migrated DB. Likely your configuration is not properly set for all the components (in this case - webserver).

As explained in https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html

> Use the same configuration across all the Airflow components. While each component does not require all, some configurations need to be same otherwise they would not work as expected. A good example for that is secret_key which should be same on the Webserver and Worker to allow Webserver to fetch logs from Worker.

**Comment by bneal1987** [comment]
The error is no longer an issue as we found out that after creating the user, we needed to initialize the database. Once that was completed, we were able to access and no further issues to report.

**Reply by Johnchristo11** [reply]
Hi, i am getting missing module'Pendulam' error while initializing the airflow.. Can you pls let the steps you followed for installing the same. thanks

**Reply by potiuk** [reply]
https://airflow.apache.org/docs/apache-airflow/stable/installation/installing-from-pypi.html
